.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+### Steps to reproduce the expected behaviour
+
+
+### Expected behaviour
```

Comment:
Would it be nice to have “actual behaviour” as well as “expected behaviour”?

Reply:
And maybe, some kind of 'Short description' at the beginning?

Reply:
Nice suggestions, thanks